be/src/exec/analytor.cpp

```diff
@@ -1187,6 +1196,7 @@ void Analytor::_find_candidate_peer_group_ends() {
 }
 
 void Analytor::_get_window_function_result(size_t frame_start, size_t frame_end) {
+    SCOPED_THREAD_LOCAL_AGG_STATE_ALLOCATOR_SETTER(_allocator.get());
```

Comment:
_get_window_function_result will assign result for each rows.
```
// DO NOT put timer here because this function will be used frequently,
    // timer will cause a sharp drop in performance.
```
Will this cause a performance regression?

Reply:
> _get_window_function_result will assign result for each rows.
> 
> ```
> // DO NOT put timer here because this function will be used frequently,
>     // timer will cause a sharp drop in performance.
> ```
> 
> Will this cause a performance regression?

No, in analytor, we just use MemHookAllocator, the logic is the same as before. there is no new timer introduced